### PR TITLE
prefix testcases with testsuite name

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- replace [testing] by [$testsuitename] in junit output
- ...
- ...

**Related issue(s)**
#579

### CODE REVIEW

**Review by clebs — APPROVED** [review]

**Comment by k15r** [comment]
please do not merge until we have confirmation from the prow guys that this will not introduce a major issue on their side

**Comment by clebs** [comment]
Please merge yourself as soon as you get the green light.

**Comment by clebs** [comment]
Hi @k15r, what is the status with this PR? Please merge it or close if not possible to merge.

**Pull request merged**
